docs/readme.md

```diff
@@ -0,0 +1,29 @@
+---
+id: readme
+title: Welcome to FAST-DNA
+sidebar_label: What is FAST-DNA
+---
+
+Check the [documentation](https://docusaurus.io) for how to use Docusaurus.
+
+## Lorem
```

Comment:
Why are we including boilerplate here?

Reply:
Per discussion to be removed in future PR